src/core.c/Hash/Object.pm6

```diff
@@ -33,6 +33,19 @@ my role Hash::Object[::TValue, ::TKey] does Associative[TValue] {
         )
     }
 
+    method PUSH_FROM_ITER(Mu \iter, \target --> Nil) {
```

Comment:
Guessing, this one has to be an implementation details too, same as on `Hash`.

Reply:
Makes sense. I was just taking STORE_AT_KEY as the basis and saw no trait on it.